installers/conf/templates/helm_template.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
It would be great to be able to keep the Helm specific stuff down in the Helm install directories (and therefore keep the `conf` dir for the config that applies to all installers).  Can we therefore move this file down into `installers/helm/postgres-operator`?

I'm thinking we can then leverage `.helmignore` to keep this file from being picked up in the event the chart is ever packaged into an archive file (e.g. via `helm package`).